src/main/java/net/masterthought/cucumber/json/deserializers/EmbeddingDeserializer.java

```diff
@@ -24,7 +24,16 @@
     @Override
     public Embedding deserialize(JsonNode rootNode, Configuration configuration) {
         String data = rootNode.get("data").asText();
-        String mimeType = rootNode.get("mime_type").asText();
+        JsonNode media = rootNode.get("media");
+        String mimeType;
+
+        if (media != null) {
+            // v3 reports have media object field with type string property
```

Comment:
`v3` is not something I like to have in code as in next month nobody will understand the purpose of such versioning :)